flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -52,6 +56,34 @@
             1,
             True,
             False,
+            False,
+            None,
+            5,
+            None,
+        ),
+        (
+            "test_supplier_user_4@seita.nl",
+            None,
+            None,
+            None,
+            None,
+            None,
+            None,
+            True,
```

Comment:
Add a small comment why you expect the error, so the reader knows, also the one below

Reply:
noted